x-pack/plugins/security_solution/public/management/components/policy_response/policy_response_friendly_names.ts

```diff
@@ -6,270 +6,412 @@
+        'xpack.securitySolution.endpoint.details.policyResponse.read_elasticsearch_config',
+        { defaultMessage: 'Read Elasticsearch Config' }
+      ),
+    ],
+    [
+      'read_events_config',
+      i18n.translate('xpack.securitySolution.endpoint.details.policyResponse.read_events_config', {
+        defaultMessage: 'Read Events Config',
+      }),
+    ],
+    [
+      'read_kernel_config',
+      i18n.translate('xpack.securitySolution.endpoint.details.policyResponse.read_kernel_config', {
+        defaultMessage: 'Read Kernel Config',
+      }),
+    ],
+    [
+      'read_logging_config',
+      i18n.translate('xpack.securitySolution.endpoint.details.policyResponse.read_logging_config', {
+        defaultMessage: 'Read Logging Config',
+      }),
+    ],
+    [
+      'read_malware_config',
+      i18n.translate('xpack.securitySolution.endpoint.details.policyResponse.read_malware_config', {
+        defaultMessage: 'Read Malware Config',
+      }),
+    ],
+    [
+      'workflow',
+      i18n.translate('xpack.securitySolution.endpoint.details.policyResponse.workflow', {
+        defaultMessage: 'Workflow',
+      }),
+    ],
+    [
+      'full_disk_access',
+      i18n.translate('xpack.securitySolution.endpoint.details.policyResponse.full_disk_access', {
+        defaultMessage: 'Full Disk Access',
+      }),
+    ],
+  ])
+);
+
+type PolicyResponseStatus = `${HostPolicyResponseActionStatus}`;
+
+const policyResponseStatuses = Object.freeze(
+  new Map<PolicyResponseStatus, string>([
+    [
+      'success',
+      i18n.translate('xpack.securitySolution.endpoint.details.policyResponse.success', {
+        defaultMessage: 'Success',
+      }),
+    ],
+    [
+      'warning',
+      i18n.translate('xpack.securitySolution.endpoint.details.policyResponse.warning', {
+        defaultMessage: 'Warning',
+      }),
+    ],
+    [
+      'failure',
+      i18n.translate('xpack.securitySolution.endpoint.details.policyResponse.failed', {
+        defaultMessage: 'Failed',
+      }),
+    ],
+    [
+      'unsupported',
+      i18n.translate('xpack.securitySolution.endpoint.details.policyResponse.failed', {
+        defaultMessage: 'Unsupported',
+      }),
+    ],
+  ])
+);
+
+const descriptions = Object.freeze(
+  new Map<Partial<PolicyResponseAction> | string, string>([
+    [
+      'full_disk_access',
+      i18n.translate(
+        'xpack.securitySolution.endpoint.details.policyResponse.description.full_disk_access',
+        {
+          defaultMessage:
+            'You must enable full disk access for Elastic Endpoint on your machine. See our troubleshooting documentation for more information',
+        }
+      ),
+    ],
+  ])
+);
+
+const linkTexts = Object.freeze(
+  new Map<Partial<PolicyResponseAction> | string, string>([
+    [
+      'full_disk_access',
+      i18n.translate(
+        'xpack.securitySolution.endpoint.details.policyResponse.link.text.full_disk_access',
+        {
+          defaultMessage: 'Documentation',
```

Comment:
@kevinlog thoughts on wording for `full_disk_access` link button?

Reply:
@joeypoon 

how about: `Full disk access needs to be enabled on this host to enable all Endpoint capabilities. See the documentation[link] for more information.`

cc @caitlinbetz

Reply:
I do agree that using the link within the description is more elegant. though, I also feel like since the design had a CTA button it felt a bit more consistent if all links were just on the button. either way works for me though.